.github/workflows/terraform.yml

```diff
@@ -0,0 +1,40 @@
+name: 'Terraform'
+
+on:
+  push:
+    branches:
+    - main
+  pull_request:
+
+jobs:
+  terraform:
+    name: 'Terraform'
+    runs-on: ubuntu-latest
+    environment: production
+    env:
+      AWS_ACCESS_KEY_ID: ${{ secrets.ACCESS_KEY_ID }}
+      AWS_SECRET_ACCESS_KEY: ${{ secrets.SECRET_ACCESS_KEY }}
+
+    defaults:
+      run:
+        shell: bash
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+
+    - name: install Terraform
+      run: wget https://releases.hashicorp.com/terraform/0.15.5/terraform_0.15.5_linux_amd64.zip && unzip terraform_0.15.5_linux_amd64.zip && export PATH=$PATH:$(pwd)
+
```

Comment:
Wouldn't it be better to use the Hashicorp's github action to install it?

https://github.com/marketplace/actions/hashicorp-setup-terraform

Reply:
I think that was only to terraform cloud, but is not. so is implemented